app/models/food.rb

```diff
@@ -0,0 +1,3 @@
+class Food < ApplicationRecord
```

Comment:
Keep in mind that Models persist in the database while POROs do not. When we are retrieving data from an API and displaying it, we usually don't need to persist that data in the database, so we just need a PORO, not a model